cilium-cli/connectivity/builder/client_egress_tls_sni.go

```diff
@@ -49,6 +49,31 @@ func clientEgressTlsSniTest(ct *check.ConnectivityTest, templates map[string]str
 			}
 			return check.ResultDefaultDenyEgressDrop, check.ResultNone
 		})
+
+	yamlFile = templates["clientEgressTLSSNIWildcardPolicyYAML"]
```

Comment:
nit: Perhaps we could add a check on the map access in case the key is not there?

Reply:
:thinking: probably we can take it separately, as all the template map usage didn't have existence check :cry: